README.md

```diff
@@ -1,2 +1,130 @@
 # py-rcsb-api
 Python interface for RCSB PDB API services at RCSB.org.
+
+## Examples
+<!-- Jupyter Notebook 1--> 
+<!-- Jupyter Notebook 2--> 
+
+This is a query in GraphQL syntax requesting the experimental method of a structure.
+```
+{
+  entry(entry_id: "4HHB") {
+    exptl {
+      method
+    }
+  }
+}
+
+```
+Data is returned in JSON format
+```json
+{
+  "data": {
+    "entry": {
+      "exptl": [
+        {
+          "method": "X-RAY DIFFRACTION"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Query Objects
+To generate and POST the same query in this package, you would create a Query object. From the object, you can access the response, get an interactive editor link, or access the arguments used to create the query.
+```python
+Query(input_ids={"entry_id":"4HHB"},input_type="entry", return_data_list=["Exptl.method"])
+```
+
+### input_ids
+
+Specifies which entry, entity, etc you would like to request data for.
+
+This can be a dictionary or in some cases, a list. Dictionaries must be passed with specific keys corresponding to the arguments required in the GraphQL schema and viewable in the [GraphiQL editor](https://data.rcsb.org/graphql/index.html). <!--#TODO: You can also check what keys are required for a given type by running `get_input_keys(type name)`.--> Dictionaries will be accepted for all types and lists will also be accepted for "plural" types (entries, assemblies, etc). Lists must be passed in PDB identifier format. 
+
+For example, these two input_ids arguments for input type polymer_entity_instances (Note plural) are equivalent.
+
+```python
+input_ids={"entry_ids":["4HHB.A","4HHB.B"]}
+input_ids=["4HHB.A","4HHB.B"]
+```
+
+For polymer_entity_instance (singular), only a dictionary will be accepted. `input_ids=["4HHB.A"]` will not be accepted.
+
+```python
+input_ids={"entry_id":"4HHB", "asym_id":"A"}
```

Comment:
add example with list ["4HHB.A"], and mention that it would construct the dictionary, or they can input the dictionary as in your example.